Package.swift

```diff
@@ -63,6 +80,11 @@ let interfaceBuildSettings: [CSetting] = [
     .unsafeFlags(["-I/usr/lib/swift"], .when(platforms: [.linux, .android])) // dispatch
 ]
 
+let swiftBuildSettings: [SwiftSetting] = [
+    .define("DEPLOYMENT_RUNTIME_SWIFT"), 
```

Comment:
It looks like we define this in Swift for all platforms but in C only for Linux - just checking, is that intended?

Reply:
It may not actually matter... `CFAvailability.h` defines it now too:

```
#ifndef DEPLOYMENT_RUNTIME_SWIFT
#define DEPLOYMENT_RUNTIME_SWIFT 1
#endif
```